### ISSUE TITLE
Installation with zef fails

### ISSUE THREAD

**Issue by Tyil** [MARTIMM/semaphore-readerswriters#7]
Trying to install this package using `zef` results in the following error:

```
===> Searching for: Semaphore::ReadersWriters
===> Testing: Semaphore::ReadersWriters:ver('0.2.6'):auth('Marcel Timmerman')
===SORRY!===
Could not find META6:ver<0.0.4..*> at line 66 in:
    /home/tyil/.zef/store/semaphore-readerswriters.git/b2d704976ba00accc8c44de92a0789270afcd5ca/lib
    /home/tyil/.perl6
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/site
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/vendor
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6
    CompUnit::Repository::AbsolutePath<66249952>
    CompUnit::Repository::NQP<64088680>
    CompUnit::Repository::Perl5<64088720>
===> Testing [FAIL]: Semaphore::ReadersWriters:ver('0.2.6'):auth('Marcel Timmerman')
Aborting due to test failure: Semaphore::ReadersWriters:ver('0.2.6'):auth('Marcel Timmerman') (use --force to override)
  in code  at /home/tyil/.perl6/sources/4B1D5A60B59D9541E13F76E0E2A2D550E0144053 (Zef::Client) line 346
  in method test at /home/tyil/.perl6/sources/4B1D5A60B59D9541E13F76E0E2A2D550E0144053 (Zef::Client) line 325
  in code  at /home/tyil/.perl6/sources/4B1D5A60B59D9541E13F76E0E2A2D550E0144053 (Zef::Client) line 497
  in sub  at /home/tyil/.perl6/sources/4B1D5A60B59D9541E13F76E0E2A2D550E0144053 (Zef::Client) line 494
  in method install at /home/tyil/.perl6/sources/4B1D5A60B59D9541E13F76E0E2A2D550E0144053 (Zef::Client) line 600
  in sub MAIN at /home/tyil/.perl6/sources/09969B10D9F270B8DBAAB9961FA7E6CDB8AD58C9 (Zef::CLI) line 139
  in block <unit> at /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/site/resources/A76807CE689F66BA252B3D85DF3EABA7AE381027 line 1
```

Running `prove -e perl6 t/` on the repository results in the following:

```
t/001-meta.t .. ===SORRY!===
Could not find META6:ver<0.0.4..*> at line 66 in:
    /home/tyil/.perl6
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/site
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/vendor
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6
    CompUnit::Repository::AbsolutePath<82060400>
    CompUnit::Repository::NQP<79929048>
    CompUnit::Repository::Perl5<79929088>
t/001-meta.t .. Dubious, test returned 1 (wstat 256, 0x100)
No subtests run 
t/100-rw.t .... ===SORRY!===
Could not find Semaphore::ReadersWriters at line 3 in:
    /home/tyil/.perl6
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/site
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6/vendor
    /home/tyil/.local/var/rakudobrew/moar-nom/install/share/perl6
    CompUnit::Repository::AbsolutePath<18010400>
    CompUnit::Repository::NQP<38592320>
    CompUnit::Repository::Perl5<38592360>
t/100-rw.t .... Dubious, test returned 1 (wstat 256, 0x100)
No subtests run 

Test Summary Report
-------------------
t/001-meta.t (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: No plan found in TAP output
t/100-rw.t  (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: No plan found in TAP output
Files=2, Tests=0,  0 wallclock secs ( 0.02 usr  0.00 sys +  0.85 cusr  0.05 csys =  0.92 CPU)
Result: FAIL
```

Output of `perl6 -v`:

```
This is Rakudo version 2017.08-155-gc3a71acb3 built on MoarVM version 2017.08.1-171-gcf95892e
implementing Perl 6.c.
```

Output of `zef info zef`:

```
- Info for: zef
- Identity: zef:ver('0.1.9'):auth('github:ugexe')
- Recommended By: /home/tyil/.perl6
Description:	 It's like [cpanm] wearing high heels with a tracksuit
Source-url:	 git://github.com/ugexe/zef.git
Provides: 34 modules
Support:
#   bugtracker:	https://github.com/ugexe/zef/issues
Depends: 0 items
```

**Comment by MARTIMM** [comment]
Hi Patrick,
I've made small changes in META6.json but could not further find any problems. I've increased the version to 0.2.6.1 but will take some time to get visible from the ecosystem.

A few things what I can think of for you to try are
* clean up the cache of zef, especially of course of this module
* reinstall or update Test::META. Maybe a newer version is available for you if you already had installed it.
* update zef. the installation I use is now version 0.1.27
